Build problem with ZSTD

I recently had to recreate my build environment from scratch. After doing so, and running the 'make' command to build Minetest, I got this error:

```
minetest/src/script/lua_api/l_util.cpp:329:15: error: ‘ZSTD_CLEVEL_DEFAULT’ was not declared in this scope
```

I've confirmed that the library is installed, /usr/include/zstd.h exists, and the build system can find it.

I thought it somewhat interesting that I can't find the constant ZSTD_CLEVEL_DEFAULT anywhere in the library header except in comments, and zstd.h doesn't include any other headers in turn (except stddef.h) so I am stumped as to where this symbol is supposed to come from. The ZSTD header does say version 1.3.3.

I changed l_util.cpp to read as follows:

```
#include <zstd.h>
#ifdef ZSTD_CLEVEL_DEFAULT
#error It exists!
#endif
```

The compiler did not remark. I'm stumped.

--- a/src/script/lua_api/l_util.cpp
+++ b/src/script/lua_api/l_util.cpp
@@ -44,6 +44,11 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/png.h"
 #include <cstdio>
 
+// only available in zstd 1.3.5+
+#ifndef ZSTD_CLEVEL_DEFAULT
+#define ZSTD_CLEVEL_DEFAULT 3
+#endif
+
 // log([level,] text)
 // Writes a line to the logger.
 // The one-argument version logs to LL_NONE.

